tests/visual/dialog.html

```diff
@@ -30,7 +30,7 @@
              aria-labelledby="titlebar-id">
 
             <div class="k-window-titlebar k-dialog-titlebar k-header" id="titlebar-id">
-                <span class="k-window-title k-dialog-title">Title</span>
+                <span class="k-window-title k-dialog-title">Title dialog type</span>
```

Comment:
The text "dialog type" is misguiding, as it has nothing to do with the actual test case. Please add comments what it actually tests, like so:

```html
<span class="k-window-title k-dialog-title">
    Title
    <!-- descendants --> g y p
    <!-- ascendant --> t d l
</span>
````

This will allow future maintainers to understand why the text is that way, and prevent accidental deletion of the markup.